src/matrices.rs

```diff
@@ -84,6 +107,19 @@ impl MatrixTrait for Matrix {
         result.ensure_valid();
     }
 
+    fn try_invert(&self) -> Result<Matrix, Status> {
+        let mut matrix = *self;
+
+        let result = unsafe {
+            ffi::cairo_matrix_invert(&mut matrix)
+        };
+
+        match result {
+            Status::Success => Ok(matrix),
+            _ => Err(result)
```

Comment:
Just for small nits like this, so just squash last commit please. For the others it's fine.